superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -145,8 +145,10 @@ def _preprocess(self):
             return False
 
         # Set the environment path.
-        if 'SB_MICRO_PATH' in os.environ:
-            os.environ['PATH'] = os.getenv('SB_MICRO_PATH', '') + os.pathsep + os.getenv('PATH', '')
+        if os.getenv('SB_MICRO_PATH'):
+            os.environ['PATH'] = os.path.join(os.getenv('SB_MICRO_PATH'), 'bin') + os.pathsep + os.getenv('PATH', '')
+            os.environ['LD_LIBRARY_PATH'] = os.path.join(os.getenv('SB_MICRO_PATH'),
```

Comment:
Have you tested that the library can be found by setting 'LD_LIBRARY_PATH'?

Reply:
It works well on my current workspace, but I'm not sure if it will work will in pipeline. But I need this pr to be merged first, and then I can test and see cublas code's status. If it has bug, I will create a new pr to fix it.